app/Services/WikibaseAPIClient.php

```diff
@@ -54,6 +55,27 @@ private function get(string $action, array $params): Response
         return $response;
     }
 
+    /**
+     * @see https://meta.wikimedia.org/wiki/User-Agent_policy
+     */
+    private function getUserAgent(): string
+    {
+        $clientName = 'Wikidata MismatchFinder';
```

Comment:
I think this should be spelled without a space – otherwise it means a product named “Wikidata” followed by another (less specific) product “MismatchFinder”. I suggest `Wikidata-Mismatch-Finder`.

Reply:
Done.